subprojects/gradle/impact-shared/src/main/java/com/avito/impact/ModifiedProjectsFinder.kt

```diff
@@ -15,14 +14,22 @@ class ModifiedProjectsFinder(
         fallbackDetector.isFallback is ImpactFallbackDetector.Result.Skip
     }
 
-    val supportedProjects: Set<Project> by lazy {
+    private val supportedProjects: Set<Project> by lazy {
         rootProject
             .subprojects
             .asSequence()
             .filter { it.isSupportedByImpactAnalysis() }
             .toSet()
```

Comment:
Does `Project` implements equals and hashCode to be in `Set`?

Reply:
You are right. I've checked and it returns identityHashCode. Will fix it.